src/maus/models/edifact_components.py

```diff
@@ -97,6 +98,46 @@ def deserialize(self, data, **kwargs) -> DataElementFreeText:
         return DataElementFreeText(**data)
 
 
+@attrs.define(auto_attribs=True, kw_only=True)
+class ValuePoolEntry:
+    """
+    A value pool entry contains the EDIFACT qualifier, a meaning (German text) and an ahb expression.
+    A value pool consists of 1 to n ValuePoolEntries.
+    The data element 3055 in UTILMD is a good example for a value pool.
+    It is used in the segments NAD+MS and NAD+MR. Its ValuePoolEntries are:
+    - (key: "9", meaning: "GS1", ahb_expression: "X")
+    - (key: "293", meaning: "DE, BDEW", ahb_expression: "X")
+    - (key: "332", meaning: "DE, DVGW", ahb_expression: "X")
+    """
+
+    #: the qualifier in edifact, might be f.e. "E01", "D", "9", "1.1a", "G_0057"
+    edifact_key: str = attr.field(validator=attrs.validators.matches_re(r"^[A-Z\d\.a-z_]+$"))
```

Comment:
Ich fände qualifier als Name glaube ich passender
```suggestion
    qualifier: str = attr.field(validator=attrs.validators.matches_re(r"^[A-Z\d\.a-z_]+$"))
```

Reply:
jemand andere meinungen? @hf-krechan nur damit ich es nicht mehramsl ändere ;)

Reply:
Um es zu unterstreichen, das ist auch die "offizielle" Bezeichnung von edi@energy und wir benutzen sie auch in unserem Sprachgebrauch aktuell. Bestes Beispiel, in deinem Docstring und im Kommentar sprichst du auch vom qualifier ;)

Reply:
aee4cb6f0eb8d645a56d97e801d75c84dfb0e81f